components/forms/select.hbs

```diff
@@ -0,0 +1,49 @@
+<div class="section--bg-white demo-section">
+
+  <form 
+    class="form" 
+    action=""
+  >
+
+    <label 
+      class="form__label required" 
+      for="location">Location
+    </label>
+    <select 
+      id="location" 
+      class="form__input__select" 
```

Comment:
See note above about element chaining in BEM class names. Could just simplify to `form__select`.